src/context/Functions/filterProducts.jsx

```diff
@@ -0,0 +1,140 @@
+export function filterProducts(state,action){
+    
+    switch(action.type){
+        case "LOW_TO_HIGH":
+            return {...state,
+                items: [...state.items].sort((a, b) =>a.price-b.price),
+                low_to_High:true,
+                High_to_low:false
+             } ;
+        case "HIGH_TO_LOW":
+            return {...state,
+                items: [...state.items].sort((a, b) =>b.price-a.price),
+                low_to_High:false,
+                High_to_low:true
+                
+              };
+        case "TSHIRT" :
+            console.log("is check is",action.isChecked)
```

Comment:
Suggestion : Logic implemented for filters can be improved. Do Check it once again!!